task/buildah/0.1/buildah.yaml

```diff
@@ -488,6 +490,10 @@ spec:
 
       cat "$(workspaces.source.path)"/image-digest | tee $(results.IMAGE_DIGEST.path)
       echo -n "$IMAGE" | tee $(results.IMAGE_URL.path)
+      {
+        echo -n "${IMAGE}@"
+        cat "$(workspaces.source.path)/image-digest"
+      } > "$(results.IMAGE_REF.path)"
```

Comment:
The Tekton result size limit is a ticking time bomb, I don't think we can add any new result to the buildah task without getting rid of BASE_IMAGES_DIGESTS https://issues.redhat.com/browse/STONEBLD-2326

The last time we added a `${repo}@${digest}` type of result, it was enough to break some builds https://github.com/konflux-ci/build-definitions/pull/654

Reply:
Or, in this case, not without getting rid of `IMAGE_URL` and `IMAGE_DIGEST`. But that would probably require a huge migration

Reply:
Let's do [STONEBLD-2326](https://issues.redhat.com/browse/STONEBLD-2326) then before this.